binding.cc

```diff
@@ -542,9 +542,9 @@ NAN_METHOD(crypto_generichash_instance) {
   unsigned long long output_length = crypto_generichash_bytes();
 
   if (info[1]->IsObject()) {
-    output_length = CLENGTH(info[1]->ToObject());
+    output_length = CLENGTH(info[1]->ToObject(v8::Isolate::GetCurrent()->GetCurrentContext()).ToLocalChecked());
```

Comment:
Looking at this code now I think it is questionable of us to silently ignore a given arg that we don't understand how to interpret instead of silently carry on. Anyway, not entirely relevant to this :)

What do you think about making this line `output_length = CLENGTH(Nan::To<v8::Object>(info[1]).ToLocalChecked());`? I don't know what the original `ToLocalChecked()` does if there is an exception pending in contrast to the new `Maybe*` types

Reply:
@davedoesdev what do you think about the above suggestion?